postgres/init.sls

```diff
@@ -172,8 +183,17 @@ postgresql-db-{{ name }}:
     - owner: {{ db.owner }}
     {% endif %}
     - user: {{ db.get('runas', postgres.user) }}
-    {% if db.get('user') %}
-    - db_user: {{ db.user }}
+    {% if db.get('db_user') %}
+    - db_user: {{ db.db_user }}
+    {% endif %}
+    {% if db.get('db_password') %}
```

Comment:
Nice additions. Specifying a connections user without password, hosts and the like doesnt make much sense...